bogota-abril-2019/css-magazine-cover/Pagina.html

```diff
@@ -0,0 +1,46 @@
+<html>
+
+<head>
+	<meta charset="utf-8" />
+	<link rel="stylesheet" type="text/css" href="styles.css" media="screen" />
+	<link href="https://fonts.googleapis.com/css?family=Candal|Vast+Shadow|Ramaraja|Duru+Sans" rel="stylesheet">
+
+</head>
+
+<body>
+	<p>Por: Camilo Andres Moreno Laverde.</p>
+	<p> imagen tomada de :
+		<a href="http://www.vibe.ng/ycee-covers-magazine/">http://www.vibe.ng/ycee-covers-magazine/</a>
+	</p>
+
+	<div class="imagen">
+
+		<header>
+			<p>December 2016</p>
+			<p>www.vibe.ng</p>
+
+			<h1>vibe<span>.ng</span></h1>
+			<h2>the rise & rise of<span>ycee</span></h2>
+		</header>
+
+		<article>
+			<section>
+				<h1>+</h1>
+				<h2>3 nigerian fasion designers</h2>

```

Comment:
Seems All this text is in the same context. Please review the use of the semantic.